pyproject.toml

```diff
@@ -72,16 +78,9 @@ docs = [
     "mkdocstrings>=0.24.1",
     "mkdocstrings-python>=1.8.0",
 ]
-extras = [
-    "lightgbm>=3.3.0",
-    # https://github.com/catboost/catboost/issues/2371
-    "catboost<1.2 ; python_version == '3.8'",
-    "catboost>=1.1 ; python_version != '3.8'",
-    "xgboost>=1.5.0",
-    "scipy>=1.4.0",
-]
```

Comment:
Indeed this is extra dependency with all packages seems not very useful. But in the future it could be nice having one optional dependency for each algorithm. If the user is just interested in using probatus with lightgbm without having to install the others sounds like a fair use case.